packages/vite/src/node/plugins/manifest.ts

```diff
@@ -22,10 +22,15 @@ export interface ManifestChunk {
 export function manifestPlugin(config: ResolvedConfig): Plugin {
   const manifest: Manifest = {}
 
-  let outputCount = 0
+  let outputCount: number
```

Comment:
I might be overthinking, but this change could lead some misalignment between type and runtime. I know it should have have after buildStart, but just feel the performance difference is too little to have this change.

Reply:
It is not about performance, the issue is that in build watch mode, the outputCount will not be reset for each build. Or are you proposing to do this differently?

Reply:
Oh I mean, it might be better to keep this line, aka the initial values for this variables. But yeah, I might just overthinking it. `buildStart` should always be called anyway, otherwise it's another more serious bug :P nvm